arclightning_web/login.html

```diff
@@ -0,0 +1,39 @@
+<!doctype html>
+<html>
+  <head>
+    <link href="https://fonts.googleapis.com/css?family=Do+Hyeon" rel="stylesheet">
+    <!--bootstrap-->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u"
+        crossorigin="anonymous">
+    <!-- Vuejs -->
+    <style>
+        .centerItem {
+            display: block;
+            margin-left: auto;
+            margin-right: auto;
+        }
+
+        .formattedText {
+            font-family: 'Do Hyeon', sans-serif;
+            font-size: 64px;
+        }
+    </style>
+</head>
+
+<body>
+    <div class="row" style="width:100%; height:100vh;">
+        <div class="col-md-6 col-md-offset-3" style="height:60vh; top: 20%;">
+	 
+<!-- On pressing submit, send  a POST request to http://hostname/api/v1/signin?username=username&password=password, so just change action url to url above to "http://hostname/api/v1/signin" -->
+      <form class="formattedText" action="http://httpbin.net/post" method="post" >
+	<h1 class="formattedText">username</h1> 
+	<input type="text" id="username" style="width: 100%; border:none; border-bottom: 2px solid black;"name="username" />
+	
+	<h1 class="formattedText">password</h1> 
+	<input type="text" id="password" style="width: 100%; border:none; border-bottom: 2px solid black;" name="password" />
```

Comment:
If you change `type="text"` to `type="password"` it will use dots to hide the user input.

Reply:
Additionally, I think it may be better if you add `placeholder="username"` and `placeholder="password"` in the `input` tags and get rid of the `<h1>` username & pw

Reply:
genius boy, thank u